scripts/results/plot_compare.py

```diff
@@ -10,12 +10,13 @@
 import numpy as np
 
 import mne
-from library.config import meg_dir, ylim
+from library.config import meg_dir, ylim, l_freq
 
 evokeds = list()
 for subject_id in range(1, 20):
     subject = "sub%03d" % subject_id
-    fname_in = op.join(meg_dir, subject, '%s-ave.fif' % subject)
+    fname_in = op.join(meg_dir, subject,
+                       '%s_highpass-%sHz-ave.fif' % (subject, l_freq))
```

Comment:
@larsoner can you make a note that here we actually just want to use the l_freq = 1.0 case. See the Makefile ... the l_freq = None should be used only up to 06-make_evoked.py

Reply:
I'm not sure about this. What if someone wanted to use `l_freq=2.` in their analysis, should this line still use a 1 Hz evoked file?

In other words, I think we should use `l_freq=config.l_freq` (and assume it's set to some non-`None` value) everywhere except where we want to compare against no highpass, in which case we should hard-code the uses of `l_freq=None`.

Reply:
We should assume that user won't change `l_freq`. Because `l_freq=None` will give weird results in many places and ICA script will crash

Reply:
Or maybe you could use the `config.py` in `results` to set an `l_freq=1`. Yeah, that would be fine ... 

There are two `config.py`s ... I know it's confusing but I couldn't figure out a better way